dockerfiles/init/manifests/codenvy.pp

```diff
@@ -303,6 +303,12 @@
   $db_schema_flyway_scripts_locations=getValue("DB_SCHEMA_FLYWAY_SCRIPTS_LOCATIONS","classpath:che-schema,classpath:codenvy-schema")
 
 ###############################
+# Rsync backup/restore configuration
+  $rsync_restore_bwlimit=getValue("RSYNC_RESTORE_BWLIMIT","7000")
```

Comment:
We are going to change backup/restore feature to respect backups of other folders and machines. I'm not sure that we will leave these properties unchanged. Are you sure that we need it in codenvy.env? What can we do if we add these properties there and then change configuration in next release?